pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -100,20 +100,7 @@ rec {
   };
 
   # data center driver compatible with current default cudaPackages
-  dc = dc_520;
-  dc_520 = generic rec {
```

Comment:
Your release notes say 'has been deprecated' , but you've removed the driver entirely.